src/api/videoRoom.ts

```diff
@@ -0,0 +1,79 @@
+import Bugsnag from "@bugsnag/js";
+import firebase from "firebase/app";
+
+import { VideoChatRequest, VideoChatRequestState } from "types/VideoRoom";
+
+export const inviteToVideoChat = async (
+  hostUserId: string,
+  hostUserLocation: string,
+  invitedUserId: string
+) => {
+  const videoRoomRequest: VideoChatRequest = {
+    hostUserId: hostUserId,
+    hostUserLocation: hostUserLocation,
```

Comment:
As called out earlier, what happens if the host moves location after this invite is sent/before it's accepted. This will cause a race condition, and send them back to the wrong place afterwards.

Reply:
The host is redirected in the video room after the invite is sent successfully. No race condition here.